orderable/tests/test_models.py

```diff
@@ -207,3 +213,13 @@ def test_changing_parent(self):
         subtask_2.task = task
         subtask_2.save()
         self.assertSequenceEqual(task.subtask_set.all(), [subtask, subtask_2])
+
+    @given(lists(integers(min_value=1), min_size=1, unique=True))
```

Comment:
> I'm not proposing using it everywhere. I think it is appropriate here because Orderable.save is very complex with a ton of possible breaking edge-cases.

OK.  Do you know what those edge cases might look like?  Is there a way to use Hypothesis to help find them if they exist, and then write explicit tests to catch them?

In your test here, Hypothesis generates a set of randomish numbers to use as `sort_order` values, but the real error/problem is the `unique_together` with `SubTask` that you described earlier.  If the edge cases are anything like that, I don't think Hypothesis is going to help us - we can't use it to generate random database schemas or anything like that.